design-reviews/recipes/high-level-design-recipe.md

```diff
@@ -0,0 +1,21 @@
+# High Level / Game Plan Design Recipe
+
+## Why is this valuable?
+
+Design at macroscopic level shows the interactions between systems and services that will be used to accomplish the project. It is intended to ensure there is high level understanding of the plan for what to build, which off-the-shelf components will be used, and which external components will need to interact with the deliverable.
+
+## Things to keep in mind
```

Comment:
One thing we should consider adding is attempt to illustrate the different personas involved in the use cases and how/which boxes are their entrypoints.

Reply:
Some other ideas could be around existing vs. net new components.  I saw some colors illustrated in your diagram but it wasn't clear if that was referring to this concept or not.

Reply:
Great feedback! Incorporated into the change